src/lib/setupContextUtils.js

```diff
@@ -1,27 +1,27 @@
+import dlv from 'dlv'
 import fs from 'fs'
-import url from 'url'
 import postcss from 'postcss'
-import dlv from 'dlv'
 import selectorParser from 'postcss-selector-parser'
+import url from 'url'
 
-import transformThemeValue from '../util/transformThemeValue'
-import parseObjectStyles from '../util/parseObjectStyles'
-import prefixSelector from '../util/prefixSelector'
-import isPlainObject from '../util/isPlainObject'
+import { corePlugins, variantPlugins } from '../corePlugins'
 import escapeClassName from '../util/escapeClassName'
+import { finalizeSelector, formatVariantSelector } from '../util/formatVariantSelector'
+import isPlainObject from '../util/isPlainObject'
+import isSyntacticallyValidPropertyValue from '../util/isSyntacticallyValidPropertyValue'
+import log from '../util/log'
 import nameClass, { formatClass } from '../util/nameClass'
+import negateValue from '../util/negateValue'
+import parseObjectStyles from '../util/parseObjectStyles'
 import { coerceValue } from '../util/pluginUtils'
-import { variantPlugins, corePlugins } from '../corePlugins'
-import * as sharedState from './sharedState'
-import { env } from './sharedState'
+import prefixSelector from '../util/prefixSelector'
 import { toPath } from '../util/toPath'
-import log from '../util/log'
-import negateValue from '../util/negateValue'
-import isSyntacticallyValidPropertyValue from '../util/isSyntacticallyValidPropertyValue'
-import { generateRules, getClassNameFromSelector } from './generateRules'
+import transformThemeValue from '../util/transformThemeValue'
 import { hasContentChanged } from './cacheInvalidation.js'
+import { generateRules, getClassNameFromSelector } from './generateRules'
 import { Offsets } from './offsets.js'
-import { finalizeSelector, formatVariantSelector } from '../util/formatVariantSelector'
+import * as sharedState from './sharedState'
+import { env } from './sharedState'
```

Comment:
Hey, please revert these changes. There's no need to change them.

Reply:
Srry, my mistake